src/CMakeLists.txt

```diff
@@ -1,549 +1,534 @@
+    dialogs/updatedialog.h
+    dialogs/updatedialog.ui
+    dialogs/versiondialog.cpp
+    dialogs/versiondialog.h
+    dialogs/versiondialog.ui
+    dialogs/passworddialog.cpp
+    dialogs/passworddialog.h
+    dialogs/passworddialog.ui
+    dialogs/masterdialog.cpp
+    dialogs/masterdialog.h
+    dialogs/tagadddialog.cpp
+    dialogs/tagadddialog.h
+    dialogs/tagadddialog.ui
+    dialogs/sharedialog.cpp
+    dialogs/sharedialog.h
+    dialogs/sharedialog.ui
+    dialogs/evernoteimportdialog.cpp
+    dialogs/evernoteimportdialog.h
+    dialogs/evernoteimportdialog.ui
+    dialogs/serverbookmarksimportdialog.cpp
+    dialogs/serverbookmarksimportdialog.h
+    dialogs/serverbookmarksimportdialog.ui
+    dialogs/orphanedimagesdialog.cpp
+    dialogs/orphanedimagesdialog.h
+    dialogs/orphanedimagesdialog.ui
+    dialogs/orphanedattachmentsdialog.cpp
+    dialogs/orphanedattachmentsdialog.h
+    dialogs/orphanedattachmentsdialog.ui
+    dialogs/scriptrepositorydialog.cpp
+    dialogs/scriptrepositorydialog.h
+    dialogs/scriptrepositorydialog.ui
+    dialogs/dictionarymanagerdialog.cpp
+    dialogs/dictionarymanagerdialog.h
+    dialogs/dictionarymanagerdialog.ui
+    dialogs/actiondialog.cpp
+    dialogs/actiondialog.h
+    dialogs/actiondialog.ui
+    dialogs/tabledialog.cpp
+    dialogs/tabledialog.h
+    dialogs/tabledialog.ui
+    dialogs/notedialog.cpp
+    dialogs/notedialog.h
+    dialogs/notedialog.ui
+    dialogs/websockettokendialog.cpp
+    dialogs/websockettokendialog.h
+    dialogs/websockettokendialog.ui
+    dialogs/imagedialog.cpp
+    dialogs/imagedialog.h
+    dialogs/imagedialog.ui
+    dialogs/attachmentdialog.cpp
+    dialogs/attachmentdialog.h
+    dialogs/attachmentdialog.ui
+    dialogs/filedialog.cpp
+    dialogs/filedialog.h
+    entities/calendaritem.cpp
+    entities/calendaritem.h
+    entities/note.cpp
+    entities/note.h
+    entities/trashitem.cpp
+    entities/trashitem.h
+    entities/notesubfolder.cpp
+    entities/notesubfolder.h
+    entities/notehistory.cpp
+    entities/notehistory.h
+    entities/notefolder.cpp
+    entities/notefolder.h
+    entities/cloudconnection.cpp
+    entities/cloudconnection.h
+    entities/tag.cpp
+    entities/tag.h
+    entities/script.cpp
+    entities/script.h
+    helpers/flowlayout.cpp
+    helpers/flowlayout.h
+    helpers/htmlentities.cpp
+    helpers/htmlentities.h
+    helpers/clientproxy.cpp
+    helpers/clientproxy.h
+    helpers/toolbarcontainer.cpp
+    helpers/toolbarcontainer.h
+    helpers/qownnotesmarkdownhighlighter.cpp
+    helpers/qownnotesmarkdownhighlighter.h
+    helpers/LanguageCache.h
+    helpers/qownspellchecker.cpp
+    helpers/qownspellchecker.h
+    helpers/codetohtmlconverter.cpp
+    helpers/codetohtmlconverter.h
+    api/noteapi.cpp
+    api/noteapi.h
+    api/tagapi.cpp
+    api/tagapi.h
+    api/scriptapi.cpp
+    api/scriptapi.h
+    libraries/diff_match_patch/diff_match_patch.cpp
+    libraries/diff_match_patch/diff_match_patch.h

```

Comment:
All the libraries would ideally be linked using `target_link_libraries`

Reply:
Yes. In time.